code/modules/surgery/organs/_organ.dm

```diff
@@ -320,26 +320,54 @@ INITIALIZE_IMMEDIATE(/obj/item/organ)
 		replacement.set_organ_damage(damage)
 
 /// Called by medical scanners to get a simple summary of how healthy the organ is. Returns an empty string if things are fine.
-/obj/item/organ/proc/get_status_text(advanced)
-	if(advanced && (organ_flags & ORGAN_PROMINENT))
-		return "<font color='#cc3333'>Harmful Foreign Body</font>"
+/obj/item/organ/proc/get_status_text(advanced, add_tooltips)
+	if(advanced && (organ_flags & ORGAN_HAZARDOUS))
+		. = "<font color='#cc3333'>Harmful Foreign Body</font>"
+		if(add_tooltips)
+			. = span_tooltip("Remove surgically.", .)
+		return .
 
 	if(organ_flags & ORGAN_EMP)
-		return "<font color='#cc3333'>EMP-Derived Failure Cascade in Progress</font>"
+		. = "<font color='#cc3333'>EMP-Derived Failure</font>"
+		if(add_tooltips)
+			. = span_tooltip("Replace surgically or wait for EMP to subside.", .)
```

Comment:
This is not the only possible solution. EMP'd organs which are locked in this state can be fixed via organ repair surgeries, no longer requiring replacement. I don't think any organ naturally removes the flag.

Reply:
oh I could've sworn hearts restarted after some time but i'll update it